webapp/src/components/reaction_stream/reaction_stream.tsx

```diff
@@ -45,21 +56,52 @@ const ReactionChip = styled.div<chipProps>`
     width: fit-content;
 
     ${(props) => props.highlight && `
-        background: #FFBC1F;
+        background: #FFFFFF;
         color: #090A0B;
   `}
 `;
 
+type VoiceUserStatuses = {
+    [key: string]: UserState,
+};
+
 // add a list of reactions, on top of that add the hands up as the top element
 export const ReactionStream = (props: Props) => {
-    const reversed = [...props.reactions];
+    const vReactions = useSelector(voiceReactions);
+    const currentUserID = useSelector(getCurrentUserId);
+
+    const statuses = useSelector(voiceUsersStatuses) as VoiceUserStatuses;
```

Comment:
Why do we need the casting here? Can we set the return type in the selector itself?